crates/oxc_semantic/src/builder.rs

```diff
@@ -752,6 +761,10 @@ impl<'a> Visit<'a> for SemanticBuilder<'a> {
         /* cfg */
 
         if let Some(continue_target) = &stmt.label {
+            // SAFETY: references to arena-allocated objects are valid for the lifetime of the arena
```

Comment:
please keep me honest here. i think this is right. but definitely needs another set of eyes

Reply:
No unsafe needed! This does the trick:

```rs
self.unused_labels.reference(continue_target.name.as_str());
```

`.as_str()` keeps the `'a` lifetime.

(ditto for the other `unsafe` above)